<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolLibrary</class>
 <widget class="QDialog" name="ToolLibrary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Library</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="comboBoxLibrary">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Library</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonAddLibrary">
           <property name="toolTip">
            <string>Add a library</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonRemoveLibrary">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonExportLibrary">
           <property name="toolTip">
            <string>Export the current library</string>
           </property>
           <property name="text">
            <string>Export Library</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButtonDeleteTool">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Delete Selected Tools</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonCreateTool">
           <property name="toolTip">
            <string>Create a new tool</string>
           </property>
           <property name="text">
            <string>Add a Tool</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Close</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QListWidget" name="listWidgetTools">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEditSearch">
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Find a tool</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidgetTools</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
